filters/auth/oidc.go

```diff
@@ -165,6 +168,21 @@ func (s *tokenOidcSpec) CreateFilter(args []interface{}) (filters.Filter, error)
 		return nil, err
 	}
 
+	subdomainsToRemove := 1
+	if len(sargs) > paramSubdomainsToRemove && sargs[paramSubdomainsToRemove] != "" {
```

Comment:
Looking at https://github.com/zalando/skipper/blob/7e92c9ab9bd7c791e4388bfcc5f73b71cee3751f/filters/auth/oidc.go#L430-L438

I am wondering if we should rather introduce `Cookie Options` similar to `Auth Code Options` to allow multiple cookie-related options, e.g. to customize path, name or domain.

Reply:
sgtm

Reply:
@szuecs Lets decide if we go for scalar parameter like proposed and add more cookie-related parameters if needed in the future or we go for extensible key-value parameters for cookie config similar to auth code options.

Reply:
I gue a map is not too complicated but I don't know what value this has. It depends on the dns setup but we can argue a map will add more freedom to the user, but it is also more complicated.
I don't know. What is your favorite?

Reply:
> What is your favorite?

Lets not overcomplicate then and have it like proposed. The oidc* filter interface is bloated already we can not improve it much anyway.